src/main/java/momo/task/Task.java

```diff
@@ -0,0 +1,44 @@
+package momo.task;
+
+/**
+ * This class represents all task objects that have been created,
+ * which have a {@code isComplete} property which can be toggled
+ * on and off via the {@code markComplete} and {@code unmark} methods
+ */
+
+public class Task {
+    String task;
+    boolean isComplete;
+
+    public Task(String task, boolean isComplete) {
```

Comment:
Consider renaming the task parameter to something more specific so that it is not confused with the Task class.